Bugsnag/Configuration/BugsnagConfiguration.m

```diff
@@ -110,7 +84,7 @@ - (nonnull id)copyWithZone:(nullable NSZone *)zone {
     [copy setEnabledReleaseStages:self.enabledReleaseStages];
     [copy setRedactedKeys:self.redactedKeys];
     [copy setMaxBreadcrumbs:self.maxBreadcrumbs];
-    [copy setMetadata: [[BugsnagMetadata alloc] initWithDictionary:[[self.metadata toDictionary] mutableCopy]]];
+    copy->_metadata = [[BugsnagMetadata alloc] initWithDictionary:[[self.metadata toDictionary] mutableCopy]];
```

Comment:
Purely to avoid redeclaring the property which would need another Class Extension added...

Reply:
To avoid declaring the property as readwrite in the header file.

A second class extension in the `.m` file cannot be used to redeclare a property (that leads to a compile error) so the choices are make it readwrite for everyone, or use this approach used here.

Reply:
Seems reasonable to me